### ISSUE TITLE
Extension cannot recognize that I am logged into Mint

### ISSUE THREAD

**Issue by Ikelume** [mitchglass97/Mint-Crypto-Chrome#3]
Regardless of what I do I continuously receive the error telling me to please log into Mint when I press the "Sync to Mint" button. This is before I've even conducted the initial setup. The issues hasn't allowed me to even get to that point. Any advice?

**Comment by mitchglass97** [comment]
yes the extension is currently broken since the latest Mint website update. I am working on an update.

**Comment by Ikelume** [comment]
Nice, I appreciate your efforts. I recently forked the repo to see if I could figure it out myself as well. My initial thought is that the `isUserLoggedIntoMint()` function is failing due to the fact that the navbar changed to a side navigation bar compared to what it was in your demo video you posted on Reddit.

**Comment by mitchglass97** [comment]
That is definitely one of the issues but I already have a fix for that. If you want to help the real issue I am trying to figure out is clicking the Save button after changing the value of the Cryptocurrency property input field. The button doesn't seem to be clickable, even after I remove the "disabled" attribute and change the "button-disabled" class to "button-enabled". That's what I'm struggling with currently. I'm afraid it may be something on the backend but am hoping there is a trick that can be found. 

I have a code change that gets to the point of trying to click the button but the button is not clickable, so I can commit that later today if you wanna check it out.

**Comment by Ikelume** [comment]
Sure that would be great!
